### PR TITLE
logstash 2.3.1 and 2.2.4

### PR DESCRIPTION
**Pull request by dvic**

### CODE REVIEW

**Comment by gerardorochin** [comment]
Please add too 2.2.4 (Y)

**Comment by dvic** [comment]
Done :+1:

**Comment by dvic** [comment]
Seems 2.2.4 is not yet published?

**Comment by tduffield** [comment]
Is there anyway we can re-run the travis? I'm pretty sure it'll pass this time! 🙏

File: 2.2/Dockerfile [hunk 1]
```diff
@@ -21,14 +21,13 @@ RUN set -x \
 	&& chmod +x /usr/local/bin/gosu \
 	&& gosu nobody true
 
-# https://www.elastic.co/guide/en/logstash/2.0/package-repositories.html
-# https://packages.elasticsearch.org/GPG-KEY-elasticsearch
-RUN apt-key adv --keyserver ha.pool.sks-keyservers.net --recv-keys 46095ACC8548582C1A2699A9D27D666CD88E42B4
+# https://www.elastic.co/guide/en/logstash/2.2/package-repositories.html
+RUN wget -qO - https://packages.elastic.co/GPG-KEY-elasticsearch | apt-key add -
```
Comment (tianon) [inline 2.2/Dockerfile]: What's the rationale behind this change?  The previous version was verifying that the fingerprint of the key we fetch actually matches what we expect (giving us that extra layer of trustworthiness), while this method only verifies the SSL connection (which is why we didn't do it this way to begin with).

Comment (dvic) [inline 2.2/Dockerfile]: Agreed, I will change this but update the link to the documentation to the corresponding versions.

**Comment by tianon** [comment]
LGTM
